java/README.md

```diff
@@ -55,3 +55,23 @@
      human-readable.
 
 9. Update the code to insert a document with the regular client. What happens?
+
+## KMIP
+
+**KMIP support in this guide is to be considered "best effort"**.
+
+This guide uses the `direnv` tool to manage environment variables. 
+
+Update the `TRUSTSTORE_PASSWORD` environment variable in your `.envrc` file to your global Java TrustStore password.
+
+Run `direnv allow` to add the `TRUSTSTORE_PASSWORD` and `KEYSTORE_PASSWORD` environment variables.
+
+Run `source ./configure_certs.sh` to download the drivers tools repo, install certificates, and start your KMIP
```

Comment:
comment: clone may be more intuitive since they may not see anything download (i.e. wget or curl)
```suggestion
Run `source ./configure_certs.sh` to clone the drivers tools repo, install certificates, and start your KMIP
```

Reply:
comment: It may be better to generalize the repo. "The drivers tools repo" is fairly meaningless but oddly specific. Maybe "a tooling repository"?